std/src/main/scala/cats/std/either.scala

```diff
@@ -0,0 +1,64 @@
+package cats
+package std
+
+trait EitherInstances extends EitherInstances1 {
+  implicit def eitherInstances[A]: Monad[Either[A, ?]] with Traverse[Either[A, ?]] =
+    new Monad[Either[A, ?]] with Traverse[Either[A, ?]] {
+      def pure[B](b: B): Either[A, B] = Right(b)
+
+      def flatMap[B, C](fa: Either[A, B])(f: B => Either[A, C]): Either[A, C] =
+        fa.right.flatMap(f)
+
+      override def map[B, C](fa: Either[A, B])(f: B => C): Either[A, C] =
+        fa.right.map(f)
+
+      def traverse[F[_]: Applicative, B, C](fa: Either[A, B])(f: B => F[C]): F[Either[A, C]] =
+        fa.fold(
+          a => Applicative[F].pure(Left(a)),
+          b => Applicative[F].map(f(b))(Right(_))
+        )
+
+      def foldLeft[B, C](fa: Either[A, B], c: C)(f: (C, B) => C): C =
+        fa.fold(_ => c, f(c, _))
+
+      def foldRight[B, C](fa: Either[A, B], c: C)(f: (B, C) => C): C =
+        fa.fold(_ => c, f(_, c))
+
+      def foldRight[B, C](fa: Either[A, B], c: Lazy[C])(f: (B, Lazy[C]) => C): Lazy[C] =
+        fa.fold(_ => c, b => Lazy(f(b, c)))
+    }
+
+  implicit def eitherOrder[A: Order, B: Order]: Order[Either[A, B]] = new Order[Either[A, B]] {
+    def compare(x: Either[A, B], y: Either[A, B]): Int = x.fold(
+      a => y.fold(Order[A].compare(a, _), _ => -1),
+      b => y.fold(_ => 1, Order[B].compare(b, _))
```

Comment:
Sorry we haven't added it to the contributor guide, but can you please follow the convention agreed upon in #27 and use `implicit A: Order[A], B: Order[B]` and then use `A.compare` and `B.compare` instead of `Order[A].compare` and `Order[B].compare`?